plugin-ci/setup/action.yaml

```diff
@@ -21,7 +21,7 @@ inputs:
     required: false
 
   golang-version:
-    default: "1.19"
+    default: "1.21"
```

Comment:
do we need to make any extra testing for this before merge and affect all the plugins CI?

Reply:
I think that due to https://go.dev/blog/compat this is not needed.

Reply:
We will make this configurable

Reply:
I thought https://github.com/mattermost/actions-workflows/pull/30 is about the `golangci-lint` version, not the `go` version.